clog/loggers.py

```diff
@@ -269,7 +269,7 @@ def close(self):
             self.stream_files[name].close()
 
     def _create_file(self, stream):
-        return open(os.path.join(config.log_dir, stream + '.log'), 'ab', 0)
+        return open(os.path.join(config.log_dir.value, stream + '.log'), 'ab', 0)
```

Comment:
`staticconf` `get_string` returns a `ValueProxy`, perhaps you're thinking of `read_...` ?